app/src/containers/StallCard/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Button } from 'components/Button';
+import { CenteredCard } from 'components/Card';
+import { Input } from 'components/Input';
+import { Checkbox } from 'components/Checkbox';
+import { FileUpload } from 'components/FileUpload';
+import { StyledBackground } from '../LoginCard/styledLoginPage';
+
+export const StallCard = () => {
+  const handlePublic = () => {
+    console.log('Change checked state');
+  };
+
+  const handleShipping = () => {
+    console.log('Change shipping state');
+  };
+
+  const handleButton = () => {
+    console.log('test');
+  };
+
+  return (
+    <StyledBackground>
+      <CenteredCard>
+        <h2>Add Stall</h2>
+        <form>
+          <br />
+          <p> City </p>
+          <Input value="City" type="text" />
+          <p> Description </p>
+          <Input value="Description" type="text" />
+          <br />
+          <br />
+          <FileUpload /> <br />
+          <br />
+          <Checkbox id="shipping" onClick={handleShipping}>
+            {' '}
+            Is Shipping Available{' '}
+          </Checkbox>
+          <br />
+          <br />
+          <Checkbox id="public" onClick={handlePublic}>
+            {' '}
+            Is Public{' '}
+          </Checkbox>
+          <br />
+          <p> Name </p>
+          <Input value="Name" type="text" />
+          <p> Shipping price </p>
+          <Input value="Shipping price" />
```

Comment:
I think such inputs like price (and maybe also phone number) shouldn't allow anyone to enter characters that aren't numerical. I think this would do the job: 
```suggestion
          <Input value="Shipping price" type="number" />
```
Please, look at this page, since I believe it may help you with different inputs: https://www.w3schools.com/html/html_form_input_types.asp
If my component doesn't accept something (like min and max values), just pass it in `Input/index.js`